soccer_ipm/config/ipm.yaml

```
@@ -0,0 +1,25 @@
+soccer_ipm:
+  ros__parameters:
+    ball:
```

For consistency with other msg types

```suggestion
    balls:
```